src/html/index.html.template

```diff
@@ -48,7 +48,7 @@
       }
       @media (prefers-color-scheme: dark) {
         html {
-          background-color: var(--primary-background-color, #111111);
+          background-color: #111111;
         }
         #ha-init-skeleton::before {
           background-color: #1c1c1c;
```

Comment:
Is there a reason why this is not also `#111111`?

Reply:
This is the header bar